django/contrib/sessions/backends/base.py

```diff
@@ -275,6 +380,20 @@ def set_expiry(self, value):
             value = value.isoformat()
         self["_session_expiry"] = value
 
+    async def aset_expiry(self, value):
+        if value is None:
+            # Remove any custom expiration for this session.
+            try:
+                await self.apop("_session_expiry")
```

Comment:
IMO, we cannot use `apop()` here as it not always set `modified = True` so it doesn't work in exactly the same way as `__delitem__`.